redis/db/redisbp/monitored_client.go

```diff
@@ -21,11 +22,19 @@ type PoolStatser interface {
 	PoolStats() *redis.PoolStats
 }
 
+func getDeploymentType(addr string) string {
+	if strings.Contains(addr, "cache.amazonaws") {
+		return "elasticache"
+	} else {
+		return "reddit"
+	}
+}
+
 // NewMonitoredClient creates a new *redis.Client object with a redisbp.SpanHook
 // attached that connects to a single Redis instance.
 func NewMonitoredClient(name string, opt *redis.Options) *redis.Client {
 	client := redis.NewClient(opt)
-	client.AddHook(SpanHook{ClientName: name})
+	client.AddHook(SpanHook{ClientName: name, Type: "standalone", Deployment: getDeploymentType(opt.Addr)})
```

Comment:
I believe that https://pkg.go.dev/github.com/go-redis/redis/v8#Options.DB and https://pkg.go.dev/github.com/go-redis/redis/v8#FailoverOptions.DB are the database label we needed (we don't have it for cluster client, so just use empty strring/"0")

Reply:
spec calls for empty string if we don't have it, but I'm updating the spanHook to pass it when we do have it